### ISSUE TITLE
New update of Django v3 and Wagtail v2.8 is not supported

### ISSUE THREAD

**Issue by oladhari** [wagtail/wagtail-review#19]
after updating my app to Django v3 and wagtail v2.8 I got this error:
```console
/.virtualenvs/reachat/lib/python3.8/site-packages/wagtail_review/models.py:14: RemovedInWagtail29Warning: send_mail has been moved from wagtail.admin.utils to wagtail.admin.mail            
  from wagtail.admin.utils import send_mail
```
is there any soon update of wagtail_review in the coming days to take in consideration these changes?
thank you

**Comment by Koed00** [comment]
Now that v2.9 is out and that module has moved, reviews is no longer compatible with v2.9.

**Comment by oladhari** [comment]
> Now that v2.9 is out and that module has moved, reviews is no longer compatible with v2.9.

there won't be any update for wagtail reviews to support these changes?

**Comment by jmbruno** [comment]
I think "Pip install wagtail-review" is setting up the wrong library.

**Comment by Vadimkatr** [comment]
Soooo, are there any plans to support versions Wagtail v2.8 and higher?

**Comment by oladhari** [comment]
I am afraid that they stopped supporting wagtail review

**Comment by oladhari** [comment]
the code has been updated on May 12 to support the new change of [deprecated imports](https://github.com/wagtail/wagtail-review/compare/fix-deprecated-imports) 
I am using wagtail-review version 0.1.1
is there any new version supporting that pull request? where I can find the history of releases?
I would like to work on wagtail-review and fix the updates to support the new versions of Django and Wagtail

**Comment by jacobtoppm** [comment]
Hi all! The current plan is that:
- This repo, Wagtail Review, will have a new compatible version released, which works using the existing Review models
- A new variant which works using the new 2.10 Workflow/Task system instead will be released under a new name

**Comment by gasman** [comment]
I've created a new branch https://github.com/wagtail/wagtail-review/tree/pre-workflow based on [the last commit before the 2.10-workflow-specific breaking changes began](41fcbc4a67d975afd1e816e20cce805fe5a00bd5), with the minimal fixes required to get it working on current versions of Wagtail - it now passes the test suite on all Wagtail versions from 2.6 to current master, although I haven't done any proper manual testing of it yet.

Based on the latest discussions, then, I'll proceed as follows:

* rename the `master` branch to `workflow`, with the intention of it being a starting point for @jacobtoppm's fork
* rename `pre-workflow` to `main` and make it the default branch. (I figure we might as well kill two birds with one stone here and get [the master -> main rename](https://github.com/github/renaming) out of the way, since it's not going to cause any more disruption than we've already done :-) )
* Finish manual testing and release a backwards-compatible version as wagtail-review 0.2

**Comment by oladhari** [comment]
> I've created a new branch https://github.com/wagtail/wagtail-review/tree/pre-workflow based on [the last commit before the 2.10-workflow-specific breaking changes began](41fcbc4a67d975afd1e816e20cce805fe5a00bd5), with the minimal fixes required to get it working on current versions of Wagtail - it now passes the test suite on all Wagtail versions from 2.6 to current master, although I haven't done any proper manual testing of it yet.
> 
> Based on the latest discussions, then, I'll proceed as follows:
> 
> * rename the `master` branch to `workflow`, with the intention of it being a starting point for @jacobtoppm's fork
> * rename `pre-workflow` to `main` and make it the default branch. (I figure we might as well kill two birds with one stone here and get [the master -> main rename](https://github.com/github/renaming) out of the way, since it's not going to cause any more disruption than we've already done :-) )
> * Finish manual testing and release a backwards-compatible version as wagtail-review 0.2

Thank you so much! looking forward to this update!

**Comment by gasman** [comment]
Have now released wagtail-review 0.2, with compatibility up to Wagtail 2.11.

**Comment by oladhari** [comment]
> Have now released wagtail-review 0.2, with compatibility up to Wagtail 2.11.

very good news! I am so thankful @gasman

**Comment by oladhari** [comment]
@gasman I am having so big troubles to update wagtail review, I am almost doing everything from start, is there any guide to explain the changes ...
besides comments are not working any more on the page, catching the respond result and comment is not possible,can not find the request.POST, data in the frontend.py view, is not rendered correctly, there is not data at all, all what I got is token with the review ID only ...
